src/integrations/feed/clients.ts

```diff
@@ -34,15 +46,18 @@ export class FeedClient implements IFeedClient {
     config,
     extraMetadata: GenericMetadata = undefined,
   ): Promise<FeedResponse> {
-    const res = await retryFetchParse<RawFeedServiceResponse>(
-      this.url,
-      {
-        ...this.fetchOptions,
-        method: 'POST',
-        body: JSON.stringify(config),
-      },
-      { retries: this.fetchOptions.retries ?? 5 },
-    );
+    const res = await this.garmr.execute(() => {
+      return retryFetchParse<RawFeedServiceResponse>(
+        this.url,
+        {
+          ...this.fetchOptions,
+          method: 'POST',
+          body: JSON.stringify(config),
+        },
+        { retries: 0 },
```

Comment:
0 retries now that Garmr handles them. If retries where left to our wrapper they would actually happen internally and retry no matter if breaker was active or not.